.github/workflows/auto.yml

```diff
@@ -0,0 +1,40 @@
+name: Automatic CI for TFE Modules
+
+on: push
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  terraform_format:
+
+    name: Run terraform fmt
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.14.7
+
+    - name: Format Public Install
+      run: terraform fmt -check -- ${{ github.workspace }}/tests/public-install
```

Comment:
We should use run this against the root module with `-recursive` so that we can check all directories in one step.